server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -187,7 +192,18 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 				<b>Preferred Language:</b> $preferredLanguage <br/>
 				<b>Email (if provided):</b> $emailAddress <br/>
 				<b>Type:</b> $appointmentType";
-			EmailUtilities::sendHtmlFormattedEmail($toEmailString, 'VITA -- Appointment Ready', $readyMessage);
+			EmailUtilities::sendHtmlFormattedEmail($toEmailString, 'VITA -- Appointment Ready', $volunteerMessage);
+
+			if(!empty($emailAddress)) {
+				// Email client confirmation
+				$clientMessage = "Congratulations, $firstName $lastName, you have successfully marked your appointment as ready!<br/>
```

Comment:
You can ignore this for now, but if you have any guidance on SMTP, I would love it.